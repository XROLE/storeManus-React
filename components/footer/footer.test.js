import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {
  it('Should render just the footer component', () => {
    const component = shallow(
      <Footer />,
    );
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
