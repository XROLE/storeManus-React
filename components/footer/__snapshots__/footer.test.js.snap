// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render just the footer component 1`] = `
<footer
  className="signin-footer"
>
  <div
    className="footerDiv"
  >
    <p
      className="copy-right"
    >
       Â© xrole 
    </p>
  </div>
</footer>
`;
